eng/Build.props

```diff
@@ -16,25 +16,25 @@
     <!-- Exclude the websockets samples for now because they use classic .csproj, which is not yet supported in our build. -->
     <ProjectToExclude Include="$(RepoRoot)src\Middleware\WebSockets\samples\**\*.csproj" />
 
-    <!-- These projects are meant to be executed by tests. -->
-    <ProjectToExclude Include="
-                      $(RepoRoot)src\Tools\dotnet-watch\test\TestProjects\**\*.csproj;
-                      $(RepoRoot)src\Tools\Tests.Common\TestProjects\**\*.csproj;
-                      $(RepoRoot)src\Razor\Razor.Design\test\testassets\**\*.*proj;
-                      $(RepoRoot)src\Razor\test\testassets\**\*.*proj;
-                      $(RepoRoot)src\Components\Web.JS\node_modules\**\*.*proj;
-                      $(RepoRoot)src\Components\WebAssembly\Sdk\testassets\**\*.csproj;
-                      " />
+    <!-- These projects are meant to be referenced only by tests. -->
+    <ProjectToExclude Include="$(RepoRoot)src\**\testassets\**\*.*proj"
+      Exclude="$(RepoRoot)src\Components\WebAssembly\testassets\WasmLinkerTest\*.*proj;
+               $(RepoRoot)src\Components\WebView\Samples\PhotinoPlatform\testassets\PhotinoTestApp\*.*proj;
+               $(RepoRoot)src\Http\Routing\test\testassets\RoutingSandbox\*.*proj;
+               $(RepoRoot)src\Security\Authentication\Negotiate\test\testassets\Negotiate.Client\*.*proj;
+               $(RepoRoot)src\Security\Authentication\Negotiate\test\testassets\Negotiate.Server\*.*proj;
```

Comment:
@BrennanConroy @captainsafia @davidfowl @JamesNK @javiercn @pranavkm @SteveSandersonMS @Tratcher you either created or updated the above projects at some point. Are all of them still relevant i.e. important to build in the CI even though no tests use them❔ I suspect the answer is "yes" for the two Negotiate projects but am doubtful about the other three.

Reply:
/ping

Does anyone know the current status of WasmLinkerTest.csproj, PhotinoTestApp.csproj, and RoutingSandbox.csproj❔

Reply:
Didn't actually mean to merge this yet☹️ Will leave it alone and we can clear out or move those projects in a follow-up if that's appropriate.

Reply:
Yes, we should keep building the Negotiate projects, if only to catch build breaks.